### PR TITLE
Improve wording on some dialogs

### LINKED ISSUE #8132: Improve dialog "Create an organization"
Some feedback regarding this dialog:
 
image

- [x] `What is your need for Parsec?` does not sounds natural to me (Franglish?). I suggest `Choose the option that best suits your needs:`

- [ ] Left image: screenshot is not well suited as is too small. We should use the Parsec Share logo instead.
- [ ] Right image: text is in French although app configured in English: We should find an alternative like using only "15" or maybe a variant of the Parsec Share logo with overlapped word "Trial"?

- [x] Left text: `Create my organization on Parsec Share (Customer account required)`
- [x] Right text: `Try Parsec Share for 15 days (No customer account required. Your data will be deleted at the end of the trial period)`.

- [x] `I have set up my own server` is not necessarily right since the user is not necessarily the one who have set up the server. Maybe `Use my own Parsec server`?

### LINKED ISSUE #8138: Improve dialog Custom server
Some feedback regarding this dialog:
 
image

- Title should match the option leading to this dialog: `Create organization on my own Parsec server` (see comment in #8132). 
  - If that seems to long, I would stick to the previous title, just `Create an organization` so the user know is still in this process. With a subtitle/description like `Please enter your organization name and Parsec server address`.

- `Organization server` is not right. Should be `Parsec server address`.

- A placeholder `parsec3://...` should be added

### LINKED ISSUE #8140: Improve dialog Join organization
Some feedback regarding this dialog:

image

- Dialog title should be `Join an organization`
- Dialog description should be `If you have received an invitation email, copy the invitation link and paste it below. Otherwise, please contact an administrator of the organization you wish to join`.

### LINKED ISSUE #8157: Improve wording on customer area
image

- Left panel: it should be `Payment methods` (and not `Billing method`).

- Payment methods: `Saved cards` (should be in plural) 

- Invoices: `Amount` instead of `Price`  (in French: `Montant` instead of `Quantité`)

### PR DESCRIPTION
**Pull request by mmmarcos**
Fix #8132
Fix #8138
Fix #8140
Fix #8157

### CODE REVIEW

File: client/src/locales/en-US.json [hunk 1]
```diff
@@ -249,8 +250,8 @@
         "cancelNo": "Resume"
     },
     "JoinByLinkModal": {
-        "pageTitle": "Join by link",
-        "pleaseEnterUrl": "Please enter the organization URL",
+        "pageTitle": "Join an organization",
+        "pleaseEnterUrl": "If you have received an invitation email, copy the invitation link and paste it below.",
```
Comment (Ironicbay) [inline client/src/locales/en-US.json:254]: The link isn't really accessible from the email (it's an `https://` that redirects to a `parsec3://` which starts the process). A standard use case for this modal would be if the inviter manually sends the invitation link to the user.

File: client/src/locales/fr-FR.json [hunk 2]
```diff
@@ -249,8 +250,8 @@
         "cancelNo": "Reprendre"
     },
     "JoinByLinkModal": {
-        "pageTitle": "Rejoindre par lien",
-        "pleaseEnterUrl": "Veuillez entrer l'URL de l'organisation",
+        "pageTitle": "Rejoindre une organisation",
+        "pleaseEnterUrl": "Si vous avez reçu un email d'invitation, copiez le lien d'invitation et collez-le ci-dessous.",
```
Comment (Ironicbay) [inline client/src/locales/fr-FR.json:254]: Same remark as above

**Review by Ironicbay — CHANGES_REQUESTED** [review]

File: client/src/locales/en-US.json (hunk 1 above)
Comment (mmmarcos) [reply client/src/locales/en-US.json:254]: The [docs](https://docs.parsec.cloud/en/stable/userguide/new_user.html#receive-invite) confused me: 
```
If clicking on the URL doesn’t work, 
you can also use the Enter a URL manually dialog.
```
So, if it's not the `https` link that should be pasted here, what do you think of: 
```suggestion
        "pleaseEnterUrl": "If you have received an invitation link, please paste it below.",
```

Comment (mmmarcos) [reply client/src/locales/en-US.json]: https://github.com/Scille/parsec-cloud/pull/8202/files#diff-816bb4380aa93aeca09643d55a0bdbf2267a990978f11caede416d210881740aR254

File: client/src/locales/fr-FR.json (hunk 2 above)
Comment (mmmarcos) [reply client/src/locales/fr-FR.json]: https://github.com/Scille/parsec-cloud/pull/8202/files#diff-a030e9cda3c11da9bd4ef518b9de81c57a78be04eedf89095a5f6b1ad2ff267dR254

File: client/src/locales/en-US.json [hunk 3]
```diff
@@ -249,8 +250,8 @@
         "cancelNo": "Resume"
     },
     "JoinByLinkModal": {
-        "pageTitle": "Join by link",
-        "pleaseEnterUrl": "Please enter the organization URL",
+        "pageTitle": "Join an organization",
+        "pleaseEnterUrl": "If you have received an invitation link, please and paste it below.",
```
Comment (Ironicbay) [inline client/src/locales/en-US.json:254]:
```suggestion
        "pleaseEnterUrl": "If you have received an invitation link, please paste it below.",
```

Comment (mmmarcos) [reply client/src/locales/en-US.json]: done

**Review by Ironicbay — APPROVED** [review]

**Pull request merged**